content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Hardening Guide - Scheduler Configuration"
```

Comment:
My impression at first glance over the title is that this page belongs to the "references" section rather than the "concepts" section.

Reply:
Will sync with sig-security-docs to see if we should move this to references. If there is an agreement, we will move the hardening guides in a separate PR.

Reply:
https://github.com/kubernetes/sig-security/issues/30#issuecomment-2365336664

Reply:
Conversation isn't resolved yet, so I've unresolved it.

Reply:
That said, I don't see guides as reference material. Common practice such as is recommended in https://docsfordevelopers.com/ calls for thinking of these as separate kinds of documentation.

Reply:
Should we considering moving hardening guides to a different place outside of concepts and references?

Reply:
I think that's a different issue / discussion. Would you like to open it, @AnshumanTripathi ?

Reply:
@sftim created https://github.com/kubernetes/website/issues/48069

Reply:
Looks great, I believe you can move the page to the best-practices folder now.

Reply:
@network-charles that would separate this hardening guide from the existing guide. I'm unsure if I should merge this first and then move the two guides together or integrate this into best practices and then move the existing guide there. 

I'm also interested in listening to the sig-security group to see if there is a consensus that hardening guides should go to the best practices section. 

Once I have a bit more clarity on this, I can take action.

Reply:
I would rather we merge this into the current "concepts" location, for consistency, and then follow up with a PR to move the entire hardening guide into "best practices".

Sig-docs folks, is that plan OK with y'all?

Reply:
Suits me @tabbysable

Reply:
Sounds good. Marking this comment as resolved for now. I will create a follow up PR to move hardening guides to best practices.